chalice/Makefile

```diff
@@ -6,5 +6,6 @@ deploy:
 	cp -R ../dss ../dss-api.yml chalicelib
 	aws secretsmanager get-secret-value --secret-id ${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/gcp-credentials.json | jq -r .SecretString > chalicelib/gcp-credentials.json
 	aws secretsmanager get-secret-value --secret-id ${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/application_secrets.json | jq -r .SecretString > chalicelib/application_secrets.json
+	aws secretsmanager get-secret-value --secret-id ${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/${ADMIN_USER_EMAILS_SECRETS_NAME} | jq -r .SecretString > chalicelib/${ADMIN_USER_EMAILS_SECRETS_NAME}
```

Comment:
GCP credentials need to exist in the file system. The same is not true of admin user emails. This line creates a json file that will, as far as I know, never be used.